enhancements/dns/coredns-egressfirewall-integration.md

```diff
@@ -549,11 +558,11 @@ also validates that a label in a DNS name does not start or end with a `-`.
 type EgressFirewallDestination struct {
 	// ..
 
-	// dnsName is the domain name to allow/deny traffic to. If this is set, cidrSelector must be unset.
+	// dnsName is the domain name to allow/deny traffic to. If this is set, cidrSelector and nodeSelector must be unset.
 	// For a wildcard DNS name, the '*' will match only one label. Additionally, only a single '*' can be
 	// used at the beginning of the wildcard DNS name. For example, '*.example.com' will match 'sub1.example.com'
-	// but won't match 'sub2.sub1.example.com'
-	// +kubebuilder:validation:Pattern=^(\*\.)?([A-Za-z0-9]([-A-Za-z0-9]*[A-Za-z0-9])?\.)*[A-Za-z0-9]([-A-Za-z0-9]*[A-Za-z0-9])?\.?$
+	// but won't match 'sub2.sub1.example.com'.
+	// +kubebuilder:validation:Pattern=`^(\*\.)?([A-Za-z0-9-]+\.)*[A-Za-z0-9-]+\.?$`
```

Comment:
where is this pattern coming from? IIUC it is different from the DNSName validation

Reply:
The pattern is from the upstream ovnk PR: https://github.com/ovn-org/ovn-kubernetes/pull/4045/files#diff-afeb50cd661258df2d05c385a5779e7e33dad7e4caf85be278b190fb1a22fd7bR84